src/components/Banner.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { Alert } from '@edx/paragon';
+import {
+  Flag,
+  MoreVert,
+  Pin,
+  VerifiedBadge,
+} from '@edx/paragon/icons';
+
+const BANNER_TYPE = {
+  closed: (message) => <Alert className="d-flex flex-row align-items-start mb-0 p-2 border border-radius-1" variant="gray" icon={MoreVert}><>{message}</></Alert>,
```

Comment:
The color of the banners for closed posts and pinned posts is `light-200` but in use it seems to be way lighter than what is in the mockup.

Reply:
The mockups often use a slightly different theme, so some differences are to be expected. Go for what the mockup suggests. 

The icon here is wrong, though, it should be `QuestionAnswer` for closed. 

Also try to keep as few styles here as possible. I think border and border radius can be avoided here. 

Here is my implementation of "endorsed" for example:

```jsx
<Alert variant="success" className="p-2 m-0 rounded-0" icon={VerifiedBadge}>
  {intl.formatMessage(messages.endorsedLabel, { endorsersRole: comment.endorsedByLabel })}
</Alert>
```

Finally, I see you've used empty brackets (`<> </>`) in a lot of places where they aren't needed.  You can think of empty brackets as an alternative to array brackets `[]`. If you have a situation where you return multiple items from some component or function they can be useful. Here is better info on them: https://reactjs.org/docs/fragments.html

Reply:
I don't really understand why we need to have all banners in one place. Each of these is a dependent on the context It's in, i.e. a comment, a post etc. and can be directly added there. 

They are not heavily reused, so there is nothing to be gained from moving them away from where they are used and in fact it makes dealing with them harder. 

Please move each banner to where it's used along with the message.